.luacheckrc

```diff
@@ -0,0 +1,31 @@
+std = "lua51+minetest"
+unused_args = false
+allow_defined_top = true
+max_line_length = 999
+
+stds.minetest = {
+	read_globals = {
+		"DIR_DELIM",
+		"minetest",
+		"core",
+		"dump",
+		"vector",
+		"nodeupdate",
```

Comment:
Does no longer exist since 0.5.0-dev or so.

Reply:
It's a copy from an other project so I will remove all of this. Thank you.